CMakeLists.txt

```diff
@@ -228,3 +229,38 @@ endif()
 if(BUILD_TESTING)
   add_subdirectory(example)
 endif()
+
+install(DIRECTORY ${CMAKE_SOURCE_DIR}/include/opentracing/
+        DESTINATION include/opentracing/
+        COMPONENT DEVEL)
+install(FILES ${CMAKE_SOURCE_DIR}/3rd_party/include/opentracing/expected/expected.hpp
+        DESTINATION include/opentracing/
+        COMPONENT DEVEL)
+install(FILES ${CMAKE_SOURCE_DIR}/3rd_party/include/opentracing/variant/variant.hpp
+        DESTINATION include/opentracing/
+        COMPONENT DEVEL)
+install(FILES ${CMAKE_SOURCE_DIR}/3rd_party/include/opentracing/variant/recursive_wrapper.hpp
+        DESTINATION include/opentracing/
+        COMPONENT DEVEL)
+install(FILES ${CMAKE_BINARY_DIR}/include/opentracing/config.h
+        DESTINATION include/opentracing/
+        COMPONENT DEVEL)
+install(FILES ${CMAKE_BINARY_DIR}/include/opentracing/version.h
+        DESTINATION include/opentracing/
+        COMPONENT DEVEL)
+install(FILES ${CMAKE_BINARY_DIR}/output/libopentracing_mocktracer.so.1.5.0
+        DESTINATION lib64/
```

Comment:
Could you use the LIB_INSTALL_DIR from https://github.com/opentracing/opentracing-cpp/pull/78 instead of hard coding lib64?

Also, we don't want to reference libraries like `${CMAKE_BINARY_DIR}/output/libopentracing_mocktracer.so.1.5.0` the so extension isn't portable and fixing the version like that will mean it breaks when we update.

Could you use the existing install commands but only with the COMPONENT added?

Reply:
Yep, my bad for hard-coding the version number in right there. I'd been using the version variables for that for everything until I switched to CPack. And yes I'll switch the lib64.